src/main/kotlin/org/acejump/session/Session.kt

```diff
@@ -204,17 +205,17 @@ class Session(private val editor: Editor) {
    * See [TagVisitor.visitPrevious]. If there are no tags, nothing happens.
    */
   fun visitPreviousTag() =
-    if (tagVisitor?.visitPrevious() == true) end() else Unit
+    if (tagVisitor?.visitPrevious() == true) end(true) else Unit
 
   /**
    * See [TagVisitor.visitNext]. If there are no tags, nothing happens.
    */
-  fun visitNextTag() = if (tagVisitor?.visitNext() == true) end() else Unit
+  fun visitNextTag() = if (tagVisitor?.visitNext() == true) end(true) else Unit
 
   /**
    * Ends this session.
    */
-  fun end() = SessionManager.end(editor)
+  fun end(jumpPerformed: Boolean) = SessionManager.end(editor, jumpPerformed)
```

Comment:
How about changing this to `fun end(jump: TaggingResult.Jump?) = SessionManager.end(editor, jump)`?